spec/features/search_message_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'Search messages' do
+  scenario 'Shows message like keyword' do
+    Message.create(message: 'blahh')
+    Message.create(message: 'test_2')
+    Message.create(message: 'rahhh')
+
+    visit('/')
+    click_button('Search')
+    fill_in('keyword', with: "test")
+    click_button('Search')
+    
+    expect(page).to have_content("test_2")
```

Comment:
Maybe you could add another expectation here which asserts that the other posts are filtered out